lib/text/vtt_text_parser.js

```diff
@@ -219,6 +221,46 @@ shaka.text.VttTextParser = class {
     return cue;
   }
 
+  /**
+   * Parses a WebVTT styles from the given payload.
+   *
+   * @param {!shaka.text.Cue} cue
+   * @param {string} payload
+   */
+  static parseCueStyles(cue, payload) {
+    try {
```

Comment:
It would be better to reduce the size of the try block to avoid catching things we shouldn't.  If you use the utility below, you shouldn't need it at all.